docs/developers/documentation/index.md

```diff
@@ -136,7 +138,7 @@ subtrees:
 
 Once you've added your document to the `docs` folder and updated the `_toc.yml`, you can preview the website
  locally by running `make docs` from the root of
-the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
+the `napari` repository (assuming you've installed the [docs prerequisites](prerequisites)).
```

Comment:
```suggestion
the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
```

I think the `#` is correct, it links to the eponymous section on line 6.

Reply:
Yes you are right, I missed it